source/includes/_insights_api.md

```diff
@@ -0,0 +1,260 @@
+              "project_type": "recurring",
+              "campaigns": {
+                "data": [
+                  {
+                    "id": "509",
+                    "type": "campaign",
+                    "attributes": {
+                      "id": 509,
+                      "status": "complete",
+                      "complete_at": "2022-07-01T19:01:18.000Z",
+                    },
+                    "links": {
+                      "results_url": "https://www.tapresearch.com/insights_api/projects/318/campaigns/509",
+                      "columns_url": "https://www.tapresearch.com/insights_api/projects/318/campaigns/509/columns"
+                    }
+                  },
+                  {
+                    "id": "510",
+                    "type": "campaign",
+                    "attributes": {
+                      "id": 510,
+                      "status": "complete",
+                      "complete_at": "2022-07-07T19:01:18.000Z",
+                    },
+                    "links": {
+                      "results_url": "https://www.tapresearch.com/insights_api/projects/318/campaigns/510",
+                      "columns_url": "https://www.tapresearch.com/insights_api/projects/318/campaigns/510/columns"
+                    }
+                  },
+                  {
+                    "id": "511",
+                    "type": "campaign",
+                    "attributes": {
+                      "id": 511,
+                      "status": "active",
+                      "complete_at": null,
+                    },
+                    "links": {
+                      "results_url": "https://www.tapresearch.com/insights_api/projects/318/campaigns/511",
+                      "columns_url": "https://www.tapresearch.com/insights_api/projects/318/campaigns/511/columns"
+                    }
+                  }
+                ]
+              }
+            }
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+Get a list of projects for the organization. Request identity/organization is derived from the `Authorization` header.
+
+**Note:** Projects that are still in draft are not returned.
+
+### HTTP Request
+
+`GET https://www.tapresearch.com/insights_api/projects`
+
+### Required Parameters
+`NONE`
+
+### Optional Parameters
+Parameter | Type | Description
+--------- | ---- | -----------
+project_type | `String` | Pass this parameter to return only projects of the specified type. Accepted values: `recurring` or `one_time`
+page | `Integer` | Response is paginated, 30 projects per page. By default this endpoint returns the first page. If you have more than 30 projects, you will need to make subsequent requests using the `page` parameter.
+
+### Response
+Response follows the [JSON:API specification](https://jsonapi.org/format/). Each resource type returns a top-level `data` key, and within that, an `attributes` field containing the data.
+
+<h3 style="margin: 30px 0 10px;">Top-Level Fields</h3>
```

Comment:
Does this work in markdown?

Reply:
Yup it does in Slate, [see here](https://github.com/slatedocs/slate/wiki/Markdown-Syntax#notes-and-warnings)

Mainly using these over the `###` helpers for spacing and linking purposes.